themes/wporg-translate-events-2024/blocks/event-attendance-mode/index.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace Wporg\TranslationEvents\Theme_2024;
+use Wporg\TranslationEvents\Translation_Events;
+
+register_block_type(
+	'wporg-translate-events-2024/event-attendance-mode',
+	array(
+		// phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.Found
+		'render_callback' => function ( array $attributes, $content, $block ) {
+			if ( ! isset( $block->context['postId'] ) ) {
+				return '';
+			}
+			$event_id = get_the_ID();
+			$event = Translation_Events::get_event_repository()->get_event( $event_id );
+			if ( ! $event ) {
+				return $event_id;
```

Comment:
This should rather be this:
```suggestion
				return '';
```

Generally, since we have this section for retrieving the event repeated across many blocks, I propose to create a utility funtion or a callback wrapper in a follow-up.